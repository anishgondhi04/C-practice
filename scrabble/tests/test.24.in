10
2
1 2 H hello
1 3 H hello
2
2 l 8
2 o 5

