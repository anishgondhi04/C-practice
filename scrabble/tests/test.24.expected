Invalid word placement: (1,3) H, hello
..........
..........
.hello....
..........
..........
..........
..........
..........
..........
..........
Place vertically at (5,2)
